### PR TITLE
numpy 2.0 support

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

**Comment by nabobalis** [comment]
The LASCO data is not being found.

File: sunkit_image/tests/test_trace.py [hunk 1]
```diff
@@ -66,15 +66,11 @@ def test_occult2_remote(image_remote, filepath_IDL):
...
-    # We know that the python code detects one point extra than the IDL code.
-    # So to test it we will remove that point.
-    coords_py = np.delete(coords_py, (1745), axis=0)
-
     # Taking all the coords from the IDL form
     coords_idl = expect[:, 1:3]
 
     # Checking all the coordinates must be close to each other
-    assert np.allclose(coords_py, coords_idl, atol=1e-0, rtol=1e-10)
+    assert np.allclose(coords_py, coords_idl, atol=1e-5)
```
Comment (Cadair) [inline sunkit_image/tests/test_trace.py:73]: why?

**Review by Cadair — APPROVED** [review]

**Comment by Cadair** [comment]
Also what is with the code coverage?

File: sunkit_image/tests/test_trace.py (hunk 1 above)
Comment (nabobalis) [reply sunkit_image/tests/test_trace.py:73]: It's not 0 anymore, its like down to 5 decimal places.

**Comment by nabobalis** [comment]
> Also what is with the code coverage?

I wish I knew.

**Pull request merged**